gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -252,6 +252,15 @@ nnstreamer_query_receive (query_connection_handle connection,
           return -EREMOTEIO;
         }
         return 0;
+      } else if (data->cmd == _TENSOR_QUERY_CMD_CLIENT_ID) {
+        /* receive client id */
+        const gsize uuid_size = 37;
+        data->client_id = g_malloc0 (uuid_size);
+        if (!query_tcp_receive (conn->socket, (uint8_t *) data->client_id,
+                uuid_size, conn->cancellable)) {
```

Comment:
Well, it might be `uuid_size - 1`. Or, is there reason to include null chacter?

Reply:
You're right.! Updated.